test_scripts/Policies/Policies_Security/Trigger_PTU_NO_Certificate/1888_1_navi.lua

```diff
@@ -0,0 +1,79 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/SmartDeviceLink/sdl_core/issues/1888
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/Policies/Policies_Security/Trigger_PTU_NO_Certificate/common')
+local runner = require('user_modules/script_runner')
+
+--[[ Local Variables ]]
+local serviceId = 7
+local appHMIType = "NAVIGATION"
+
+--[[ General configuration parameters ]]
+config.application1.registerAppInterfaceParams.appName = "server"
+config.application1.registerAppInterfaceParams.appID = "SPT"
+config.application1.registerAppInterfaceParams.appHMIType = { appHMIType }
+
+--[[ Local Functions ]]
+local function ptUpdate(pTbl)
+  local filePath = "./files/Security/client_credential.pem"
+  local crt = common.readFile(filePath)
+  pTbl.policy_table.module_config.certificate = crt
+  pTbl.policy_table.app_policies[common.getAppID()].AppHMIType = { appHMIType }
+end
+
+local function startServiceSecured(pData)
+  common.getMobileSession():StartSecureService(serviceId)
+  common.getMobileSession():ExpectControlMessage(serviceId, pData)
+
+  local handshakeOccurences = 0
+  if pData.encryption == true then handshakeOccurences = 1 end
+  common.getMobileSession():ExpectHandshakeMessage()
+  :Times(handshakeOccurences)
+
+  common.delayedExp()
```

Comment:
@dboltovskyi I would propose to set delay time explicitly

Reply:
@aderiabin If the value of timeout is not defined default one (2000 ms) is used